classes/article/ArticleDAO.inc.php

```diff
@@ -182,11 +183,14 @@ function updateObject($article) {
 		);
 
 		$this->updateLocaleFields($article);
+		$contextId = Request::getContext()->getId();
```

Comment:
Better still, the Request object shouldn't be used in the DAOs and DataObjects at all -- that's kind of mixing the Controller with the Model. Isn't the `context_id` (`journal_id`) available as a column here already?